### PR TITLE
feat: Adds the web terminal component

### PR DESCRIPTION
**Pull request by AlexanderMelox**
Contributes to #70 

This adds the web terminal component.

#### Changelog

**New**

- packages/experimental/src/components/WebTerminal/WebTerminal.js 
- packages/experimental/src/components/WebTerminal/WebTerminal.mdx
- packages/experimental/src/components/WebTerminal/WebTerminal.stories.js
- packages/experimental/src/components/WebTerminal/_index.scss
- packages/experimental/src/components/WebTerminal/_storybook-styles.scss
- packages/experimental/src/components/WebTerminal/_web-terminal.scss
- packages/experimental/src/components/WebTerminal/index.js
- packages/experimental/src/components/WebTerminal/preview-components/Navigation.js
- packages/experimental/src/components/WebTerminal/preview-components/index.js
- packages/experimental/src/components/_index.scss

#### Testing / Reviewing

You should be able to click on the terminal icon in the navigation and the web terminal should appear on the right side of the screen. You should be able to close the web terminal with the "x" icon.

##### New production dependencies

No new production dependencies.

### CODE REVIEW

**Comment by AlexanderMelox** [comment]
Still need to add unit test

**Comment by AlexanderMelox** [comment]
Tests will be added in a later PR. 

Issue #358 -> Adds test in a later version

File: packages/experimental/src/components/WebTerminal/WebTerminal.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+      // Passes the event object incase the developer wants to event.preventDefault() the link redirect
+      onClick(event);
+    }
+  }, []);
+
+  const showDocumentationLinks = useMemo(() => documentationLinks.length > 0, [
+    documentationLinks,
+  ]);
+
+  return open ? (
```
Comment (davidmenendez) [inline packages/experimental/src/components/WebTerminal/WebTerminal.js:29]: i think it's a little cleaner to do something like this
```
if (!open) return;
return <div>...</div>
```
this way you don't have to wrap everything in the if statement

**Review by davidmenendez — CHANGES_REQUESTED** [review]
looks good. just one comment

File: packages/experimental/src/components/WebTerminal/WebTerminal.js (hunk 1 above)
Comment (AlexanderMelox) [reply packages/experimental/src/components/WebTerminal/WebTerminal.js:29]: awesome, I had it like that at first and changed it to this. I'll make the change

**Review by davidmenendez — APPROVED** [review]

**Pull request merged**